components/form/prompt-input/PromptInput.es6.js

```diff
@@ -3,18 +3,44 @@ import PropTypes from 'prop-types'
 import classnames from 'classnames'
 
 function PromptInput ({ className, children }) {
-
   let textarea = null
```

Comment:
Sorry, one last comment, could you try this:
```suggestion
  let textarea = child?.type?.displayName === 'PromptInputTextarea' ?? null
  (the same goes to the actions and overlay component)
```
so we can delete identifyChildComponent and switch case below. I feel nothing bad should happen if we try to render {null}, could you check?

Reply:
Refactored code and applied the suggestion